EmployeeLib/Manager.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace EmployeeLib
 {
-    public class Manager : Employee
+    public class Manager : Managed, IPerformanceReviewer
```

Comment:
assume the manager does not have a manager, just a regular employee does have a manager

Reply:
based on the initial project, a manager does have a manager, CEO was the only role that should not have a manager
I will fix it anyways

Reply:
This is a separate project. these changes are meant to try and implement specific areas in code to apply LSP.